presto-docs/README.md

```diff
@@ -0,0 +1,73 @@
+# presto-docs - Presto Documentation
+
+The presto-docs module contains the reference documentation for Presto.
+
+## Tools
+
+The engine used to create the documentation in HTML format is the Python-based
+[Sphinx](https://www.sphinx-doc.org). Installation instructions for various OS
+are [available on the Sphinx site](https://www.sphinx-doc.org/en/master/usage/installation.html).
```

Comment:
It would be nice to mention python 2 vs 3.
I guess there may be multiple ways to install sphinx (eg with or without pip), so _besides_ refering to their manual, it could be nice to have short summary how the installation may look like.

Reply:
I disagree. To make any reasonable short summary you have to assume OS of the user. You also then partially own the documentation of this and it already is maintained correctly and exists with the sphinx project. It should not be too much to ask a user to follow the link.

Also btw. that is why I added the Known Issues section - I first used apt-get and it installed an older version of sphinx on my LTS install. Similar problems will exist with brew potentially and whatever we document will be out of date quickly.

We could even so far as to de-emphasize the whole sphinx install since it is actually not needed due to the Maven build being able to do it all and it being the authoritative build on Travis and in general also for release process and such. 

If desired I can make that change.